pandas/plotting/_core.py

```diff
@@ -2443,7 +2443,7 @@ def hist_frame(data, column=None, by=None, grid=True, xlabelsize=None,
 
 def hist_series(self, by=None, ax=None, grid=True, xlabelsize=None,
                 xrot=None, ylabelsize=None, yrot=None, figsize=None,
-                bins=10, **kwds):
+                bins=10, equal_bins=False, **kwds):
```

Comment:
I'm trying the alternative approach here. `equal_bins` will be unused within the function. This is just to avoid popping `kwds`.

Reply:
Might have missed the compat issue but if you are using `.get` it shouldn't matter whether this exists as a keyword or not

Reply:
`kwds` can only contain `matploltib.pyplot.hist` input args (b/c of line 2504) and throws an error if `equal_bins` is a keyword.